components/modal/_modal.scss

```diff
@@ -65,18 +65,21 @@
     &.sdc-modal-type-custom {
       padding: 0px;
       border-top: none;
-      .sdc-modal__header {
+      .sdc-custom__header {
```

Comment:
why was a new class needed? could specificity of original class not be used?

Reply:
Because it's two different headers at all and for covering customization we suppose to add new class.

Reply:
@islavi we closed it with @ArielDKenan on telephone talk.